open_discussions/views.py

```diff
@@ -19,6 +19,11 @@
 
 def index(request, **kwargs):  # pylint: disable=unused-argument
     """Render the react app"""
+    if request.META.get("HTTP_USER_AGENT", "").startswith("facebookexternalhit"):
```

Comment:
Doing this because Facebook's crawler doesn't run any JS, so the metadata tags needed to be added to a Django html template for Facebook to recognize them. It's an option just to add these metadata tags in `index.html` and not worry about the user agent, but my thinking was that (a) we might be adding more detailed social metadata tags for specific routes in the future (#1879), and (b) if they were added to `index.html` it would be somewhat misleading to have social metadata tag values that 'disagreed' with other metadata tags. For example, the `og:url` value would not match the `<link rel="canonical" />` tag value in many cases, the `<meta name="description" />` tag would not match the `<meta property="og:description" />` tag, etc. I also did some google fu to see how other people have addressed this in SPAs and saw a few other folks who solved this by inspecting the UA.